src/app/server/Server.h

```diff
@@ -303,16 +303,35 @@ class Server
             return CHIP_NO_ERROR;
         };
 
-        void OnFabricDeletedFromStorage(CompressedFabricId compressedId, FabricIndex fabricIndex) override
+        void OnFabricDeletedFromStorage(FabricTable & fabricTable, FabricIndex fabricIndex) override
         {
-            (void) compressedId;
+            (void) fabricTable;
             auto & sessionManager = mServer->GetSecureSessionManager();
             sessionManager.FabricRemoved(fabricIndex);
 
+            // Remove all CASE session resumption state
```

Comment:
This is a drive-by fix not core to the changes here, right?  That's fine, if so, but I want to make sure I am understanding the situation correctly.